zebra/interface.c

```diff
@@ -2034,13 +2034,13 @@ DEFUN (link_params_enable,
 	/* This command could be issue at startup, when activate MPLS TE */
 	/* on a new interface or after a ON / OFF / ON toggle */
 	/* In all case, TE parameters are reset to their default factory */
-	if (IS_ZEBRA_DEBUG_EVENT)
```

Comment:
I'm a bit concerned putting this under *only* `IS_ZEBRA_DEBUG_MPLS`

are we expecting TE commands to only be MPLS?

I actually think this should be an `IS_ZEBRA_DEBUG_EVENT || IS_ZEBRA_DEBUG_MPLS` to cover both debug types.

Reply:
done